src/bookshelf/src/bksw_sec.erl

```diff
@@ -21,123 +22,405 @@
 -module(bksw_sec).
 
 -export([is_authorized/2]).
+-ifdef(TEST).
+-compile([export_all, nowarn_export_all]).
+-endif.
 
--define(SECONDS_AT_EPOCH, 62167219200).
+% is the necessary?  try removing.
 -include("internal.hrl").
 
 %%===================================================================
 %% API functions
 %%===================================================================
-is_authorized(Req0, #context{auth_check_disabled=true} = Context) ->
-    {true, Req0, Context};
-is_authorized(Req0, #context{} = Context) ->
+is_authorized(Req,  #context{auth_check_disabled=true} = Context) ->
+    {true, Req, Context};
+is_authorized(Req0, #context{                        } = Context) ->
```

Comment:
Replace #context{                                                 } with #context{}

Reply:
The spaces make it line-up with the #context{ } in the clause above.  I prefer to line things up like this when I can - makes it easier for me to see the structure in the code.  Would you like me to remove it and format it in 'normal' style?

Reply:
Will do before merge.

Reply:
Done.